editor/scene_tree_dock.cpp

```diff
@@ -3394,7 +3394,11 @@ List<Node *> SceneTreeDock::paste_nodes(bool p_paste_as_sibling) {
 	}
 
 	EditorUndoRedoManager *ur = EditorUndoRedoManager::get_singleton();
-	ur->create_action(vformat(p_paste_as_sibling ? TTR("Paste Node(s) as Sibling of %s") : TTR("Paste Node(s) as Child of %s"), paste_parent->get_name()), UndoRedo::MERGE_DISABLE, EditorNode::get_singleton()->get_edited_scene());
+	if (paste_parent) {
+		ur->create_action(vformat(p_paste_as_sibling ? TTR("Paste Node(s) as Sibling of %s") : TTR("Paste Node(s) as Child of %s"), paste_parent->get_name()), UndoRedo::MERGE_DISABLE, edited_scene);
+	} else {
+		ur->create_action(TTR("Paste Node(s) as the Root of this scene"), UndoRedo::MERGE_DISABLE, edited_scene);
```

Comment:
```suggestion
		ur->create_action(TTR("Paste Node(s) as the Root of This Scene"), UndoRedo::MERGE_DISABLE, edited_scene);
```
As much as I dislike this style, we use Title Case for UndoRedo actions.